packages/wdio-config/src/constants.ts

```diff
@@ -7,7 +7,7 @@ export const DEFAULT_CONFIGS: () => Omit<Options.Testrunner, 'capabilities'> = (
     specs: [],
     suites: {},
     exclude: [],
-    outputDir: undefined,
+    outputDir: '.wdio-artifacts/logs',
```

Comment:
I wonder if we should call the directory just `wdio` and maybe start recommending using this for all WebdriverIO related files, e.g. logs, baselines and config files

Reply:
This makes sense to me. I would have one request which would be to allow users to store wdio logs to stdout __and__ in stdout at the same time. AFAIK right now users can only decide between either of them and I think it makes sense to allow the scenario to make both possible.

Background: I personally like to have these WebDriver commands prinited in stdout so I can immediately debug them as my test go on and I print stuff in the console. If we set a default `outputDir` I would need to comment in/out the output dir option.

I think I would be fine if I can disable the `outputDir` option by just passing in a `--no-outputDir` but not sure if yargs supports recognises this and overwrites the hard coded property in the config with `undefined`. Alternatively I can imagine to pass in a `--stdout-logging` or just `--stdout` to enable both.